### ISSUE TITLE
Clear field icon inside the search box

### ISSUE THREAD

**Issue by etanot** [searx/searx#1788]
Type: Feature request

### Description
There is no `Clear Field` icon present inside the search box.

### Use Case
It's very difficult to clear text field on mobile devices, we've to either long or constantly press backspace key to clear text field. It'd be nice if search box contains a `Clear Field` icon, when text present in the box, which clears the text field in a single click. This feature would be also helpful for desktop users, because they've to also either select all text (by pressing `ctrl+a` or triple clicking using mouse) then press backspace or long press backspace.

![ddg-search-box](https://user-images.githubusercontent.com/28824347/71411696-5987f200-2642-11ea-9786-e625309b0fe5.png)

Fig 1: DDG search box contains a clear text field (cross) icon

**Comment by return42** [comment]
Good point .. if I not complete wrong, it shouldn't be hard to implement .. do you have time to send us a PR?

**Comment by etanot** [comment]
Sorry, I'll be very busy this month, may be I'll send PR next month; meanwhile if someone else wants work on this feature request, you're welcome :)
